pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -1182,7 +1182,7 @@ func TestChangefeedColumnDropsOnTheSameTableWithMultipleFamilies(t *testing.T) {
 		})
 	}
 
-	cdcTest(t, testFn, feedTestOmitSinks("sinkless"))
+	cdcTest(t, testFn, feedTestForceSink("sinkless"))
```

Comment:
I don't think we need to forceSink.. We want to run this test against all available sinks.

I think the logic is correct; I would recommend running this and other tests under stress... 
```
 ./dev test pkg/ccl/changefeedccl --filter=TestChangefeedBackfillCheckpoint --stress   
```

(Extra bonus if you decide to setup GCE worker, so that you can build run roachprod stress -- that's probably an overkill as long as you run at least few hundred iterations and verify that you've ran sinkless sufficient number of times).